src/components/a-form-label/_preview.html

```diff
@@ -0,0 +1,3 @@
+<axa-form-label htmlfor="id">Test Label</axa-form-label>
```

Comment:
i like the idea of splitting label and input. But my option if we do it then we should also provide to bind the label with name and not just id. Generally is better have name as binding in order to avoid incorrect HTML in case of id duplication